### PR TITLE
Add -quiet option reduce log spam from exechealthz

### PR DESCRIPTION
**Pull request by therc**
See kubernetes/kubernetes [#21285](https://github.com/kubernetes/kubernetes/issues/21285)

cc @bprashanth

### CODE REVIEW

File: exec-healthz/exechealthz.go [hunk 1]
```diff
@@ -141,7 +144,9 @@ func main() {
 }
 
 func healthzHandler(w http.ResponseWriter, r *http.Request) {
-	log.Printf("Client ip %v requesting /healthz probe servicing cmd %v", r.RemoteAddr, *cmd)
+	if !*quiet {
```
Comment (bprashanth) [inline exec-healthz/exechealthz.go]: can you just create a global log function that centralizes *quiet checking?

**Comment by bprashanth** [comment]
LGTM but for the nit

**Comment by therc** [comment]
Done

**Comment by bprashanth** [comment]
LGTM, thanks. Will merge when travis is green.

**Comment by therc** [comment]
Yay! Is there going to be an official new image?

**Comment by bprashanth** [comment]
@therc I just pushed out gcr.io/google_containers/exechealthz:1.1

**Pull request merged**